vertx-web/src/main/java/io/vertx/ext/web/impl/RoutingContextImpl.java

```diff
@@ -442,6 +484,43 @@ private MultiMap getQueryParams() {
     return bodyEndHandlers;
   }
 
+  private Map<Integer, Handler<Void>> getExceptionHandlers() {
+    if (exceptionHandlers == null) {
+      // order is important we we should traverse backwards
+      exceptionHandlers = new TreeMap<>(Collections.reverseOrder());
+      response().exceptionHandler(v -> exceptionHandlers.values().forEach(handler -> handler.handle(null)));
```

Comment:
Should we wrap the for each lambda with a try catch, so, in case of a user error at the handler level, the remaining handlers also get called?